Packs/Reco/Integrations/Reco/Reco.py

```
@@ -0,0 +1,320 @@
+                    "filters": {"filters": []},
+                },
+            }
+        }
+        if risk_level:
+            params["getTableRequest"]["fieldFilters"]["filters"]["filters"].append(
+                {"field": "risk_level", "stringEquals": {"value": risk_level}}
+            )
+        if source:
+            params["getTableRequest"]["fieldFilters"]["filters"]["filters"].append(
+                {"field": "data_source", "stringEquals": {"value": source}}
+            )
+        if before:
+            params["getTableRequest"]["fieldFilters"]["filters"]["filters"].append(
+                {
+                    "field": CREATED_AT_FIELD,
+                    "before": {"value": before.strftime("%Y-%m-%dT%H:%M:%SZ")},
+                }
+            )
+        if after:
+            params["getTableRequest"]["fieldFilters"]["filters"]["filters"].append(
+                {
+                    "field": CREATED_AT_FIELD,
+                    "after": {"value": after.strftime("%Y-%m-%dT%H:%M:%SZ")},
+                }
+            )
+
+        demisto.debug(f"params: {params}")
+        try:
+            response = self._http_request(
+                method="PUT",
+                url_suffix="/incident",
+                data=json.dumps(params),
+                timeout=RECO_API_TIMEOUT_IN_SECONDS,
+            )
+            if response.get("getTableResponse") is None:
+                demisto.info(f"got bad response, {response}")
+            else:
+                demisto.info(
+                    f"Count of incidents: {response.get('getTableResponse').get('totalNumberOfRecords')}"
+                )
+                alerts = (
+                    response.get("getTableResponse", {}).get("data", {}).get("rows", [])
+                )
+                demisto.info(f"Got {len(alerts)} alerts")
+        except Exception as e:
+            demisto.error(f"Findings Request ReadTimeout error: {str(e)}")
```

I think that it will be better to raise the error so users will know that it fails.
This won't prevent the fetch from keeping triggering again after the configured time

---

Dobe